explainaboard/metrics/accuracy.py

```diff
@@ -46,6 +66,86 @@ def calc_stats_from_data(
             )
         )
 
+    def _calc_metric_from_aggregate(
+        self,
+        agg_stats: np.ndarray[tuple[int], Any] | np.ndarray[tuple[int, int], Any],
+    ) -> np.ndarray[tuple[()], Any] | np.ndarray[tuple[int], Any]:
+        """Inner function of calc_metric_from_aggregate."""
+        if agg_stats.shape[-1] == 1:
+            return agg_stats.squeeze(-1)
+        return agg_stats
+
+    def calc_metric_from_aggregate(
```

Comment:
This function is prohibited to be overridden.
https://github.com/neulab/ExplainaBoard/blob/f4c48e5f54418f0a5c4df10f93be43e80c986225/explainaboard/metrics/metric.py#L452-L453